### ISSUE TITLE
Add integration test for block error

### ISSUE THREAD

**Issue by cmdcolin** [GMOD/jbrowse-components#1776]
If a block rendering error occurs

```
diff --git a/plugins/linear-genome-view/src/BaseLinearDisplay/models/serverSideRenderedBlock.ts b/plugins/linear-genome-view/src/BaseLinearDisplay/models/serverSideRenderedBlock.ts
index 79042b986..8a92c2bdf 100644
--- a/plugins/linear-genome-view/src/BaseLinearDisplay/models/serverSideRenderedBlock.ts
+++ b/plugins/linear-genome-view/src/BaseLinearDisplay/models/serverSideRenderedBlock.ts
@@ -202,6 +202,8 @@ function renderBlockData(self: Instance<BlockStateModel>) {
     )
   }

+  throw new Error('this is a failure that happens on any block render')
+


```

Then the error is not handled properly, it goes into renderBlockEffect with props undefined and gives an error on destructuring an undefined object

**Comment by cmdcolin** [comment]
Note that this may go back to this commit b7e1085e3

This is may be a misunderstanding of how makeAbortableReaction error handling works, it is unexpected that it goes into the renderBlockEffect code

**Comment by cmdcolin** [comment]
This was quick-fixed, I suggest leaving this open to get a proper integration test for the error case...retitled to reflect

**Comment by cmdcolin** [comment]
Fixed 875d0bd
